src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends FigCol {
```

Comment:
The class name 'Circle' is appropriate for a shape, but ensure that you're not using an abstract class to set behavior for it. If 'FigCol' is an abstract class, consider whether it should be an interface or a regular class based on its purpose.